src/components/Button/index.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import StyledButton from "./styled";
+
+const Button = ({children, onClick}) => {
+    return <StyledButton onClick={onClick}>
+                {children}
+            </StyledButton>
+}
```

Comment:
Jeśli chcesz mieć komponent Button, który będzie zawierał tylko style, to lepiej to zrobić tak, że w index.js wstawiasz to co masz obecnie w styled.js i importujesz komponent jako style. Takie rozwiązanie ma dwie zalety, po pierwsze, mniej zbędnego kodu, bo teraz index.js nic nie robi tak na prawdę, po drugie, jak będziesz chciał w aplikacji gdzieś sobie użyć styli z Button, to jeśli ten komponent będzie tylko opisywał style, to można spokojnie użyć w styled components w ten sposób: const RedButton = styled(Button)` color: red; `; w obecnej wersji tego komponentu, czyli jak w index.js jest jakiś return komponentu, to nie da się nadpisać styli czegoś takiego i trzeba kombinować. Ale się rozpisałem ^^